src/tests/api/BavUnhappyPath.test.ts

```diff
@@ -25,19 +26,32 @@ describe("/session Unhappy Path", () => {
         expect(sessionResponse.status).toBe(400);
         expect(sessionResponse.data).toBe("Bad Request");
     });
+});
+
+describe("/userinfo Unhappy Path - ", () => {
+    it.skip("E2E Unhappy Path Journey - Non-bearer type authentication", async () => {
+        // const userInfoResponse = await userInfoPost("Basic " + tokenResponse.data.access_token);
+        const userInfoResponse = await userInfoPost("Basic eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImYzODA1Y2UyLWQzNjYtNDY3ZC1iYzhkLWMzMTc4MGFmNTQyYyJ9.eyJzdWIiOiJhNzhjYTE2Yi1mNTQyLTQzNzgtOWNkMC1kMGFhMTc1OTYxY2QiLCJhdWQiOiJodHRwczovL3Jldmlldy1iYXYuZGV2LmFjY291bnQuZ292LnVrIiwiaXNzIjoiaHR0cHM6Ly9yZXZpZXctYmF2LmRldi5hY2NvdW50Lmdvdi51ayIsImV4cCI6MTY5OTM3MjIwN30.aD-ANk6ITSjD4wQ8t9a9Qq8TgW5zXg2pxRYF5qPGK3kCqfWrHo9g_sU35rX8XFeST2BenBHoYkMZ-C7t4k1oiw");
+        expect(userInfoResponse.status).toBe(401);
+    });
 
+    it.skip("E2E Unhappy Path Journey - Expired bearer token", async () => {
+        // Hardcoded expired Bearer Token
+        const userInfoResponse = await userInfoPost("Bearer eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImYzODA1Y2UyLWQzNjYtNDY3ZC1iYzhkLWMzMTc4MGFmNTQyYyJ9.eyJzdWIiOiJhNzhjYTE2Yi1mNTQyLTQzNzgtOWNkMC1kMGFhMTc1OTYxY2QiLCJhdWQiOiJodHRwczovL3Jldmlldy1iYXYuZGV2LmFjY291bnQuZ292LnVrIiwiaXNzIjoiaHR0cHM6Ly9yZXZpZXctYmF2LmRldi5hY2NvdW50Lmdvdi51ayIsImV4cCI6MTY5OTM3MjIwN30.aD-ANk6ITSjD4wQ8t9a9Qq8TgW5zXg2pxRYF5qPGK3kCqfWrHo9g_sU35rX8XFeST2BenBHoYkMZ-C7t4k1oiw");
```

Comment:
Our repo is public so we should not be committing any code that contains references to Access or Bearer tokens. Please reach out to Madan and get any tokens you need to use in tests added as a GitHub secret